### PR TITLE
Quiz2

### PR DESCRIPTION
**Pull request by helanan**

### CODE REVIEW

**Comment by helanan** [comment]
finished everything @zoeames

File: index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<head>
+	<title>GrowYourOwnChristmasTree.com</title>
+</head>
+
+<body>
+  <header>Grown Your Own Christmas Tree!</header><br>
+
+<form name="growtree">
+  <p>Tree Height:</p>
+  <input type="text" id="height" name="height"><br>
```
Comment (williamhickok11) [inline index.html]: This will always be a number. Change type from "text" to "number"

File: pinetree.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+};
+
+var nameInput = document.getElementById("height");
+var btn = document.getElementById("btn");
+btn.addEventListener("click", validation);
+
+function validation(){
+var heightbtn = document.getElementById("height").value;
+var shapebtn = document.getElementById("shape").value;
+if (heightbtn === "" && shapebtn === "") {
```
Comment (williamhickok11) [inline pinetree.js]: && (and) needs to be || (or). The and only fires when BOTH are empty. But we need to alert the user when either of them are empty.

**Comment by helanan** [comment]
Finished Quiz, tree grows!! YAYYYY!

**Comment by williamhickok11** [comment]
Awesome! 👍

**Comment by helanan** [comment]
Finished quiz, approved and now merging to master branch

**Pull request merged**